statemachine/contrib/diagram.py

```diff
@@ -102,7 +104,8 @@ def _get_condition_repr(cond):
             name = getattr(cond.func, "__name__", cond.func)
             return name if cond.expected_value else "!{}".format(name)
 
-        cond = ", ".join([_get_condition_repr(cond) for cond in transition.cond])
+        cond = ", ".join([_get_condition_repr(cond)
+                         for cond in transition.cond])
```

Comment:
nitpick: These changes should be reverted to the library style (`line-length = 99`). If you run the configured linters they'll be automatically reverted.